src/init/starship.ps1

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env pwsh
+
+# Starship assumes UTF-8
+[Console]::OutputEncoding = [System.Text.Encoding]::UTF8
+function global:prompt {
+    $out = $null
+    $jobs = (Get-Job | Where-Object { $_.State -eq 'Running' }).Count
```

Comment:
Unsolicited (sorry!) but FWIW, this breaks on my mac when `Get-Job | Where-Object { $_.State -eq 'Running' }` returns one or fewer results.

```Suggestion
$jobs = if ($j = Get-Job | Where-Object { $_.State -eq 'Running' }) { $j.Count }
```
or

```Suggestion
Get-Job | Where-Object { $_.State -eq 'Running' } | Measure-Object | Select-Object -Expand Count
```

Reply:
Please try again with the newest commit